### DISCUSSION TITLE
How would I go about re-enabling hibernation?

### DISCUSSION THREAD

**Discussion by YoYoBobbyJoe** [LeDragoX/Win-Debloat-Tools#162] [Q&A]
After performing the tweaks, they removed all mention of hibernation from my system. First question: Why? Hibernation is definitely not a bloat; it's a very useful tool. Second, how do I get it back? Like, it removed it from my power plan options and everything. I can't do the checkbox reenable because it's no longer there. My laptop no longer hibernates after sitting for a couple of hours, which is something I relied on.

**Comment by LeDragoX** [comment]
Can you try the last update on Main/Dev atm and see if the hibernation comes back? I'll only be satisfied after knowing it's working.
